backend/app/Chat/User/User.php

```
@@ -110,12 +112,16 @@ public static function register(string $username, string $password, string $emai
              */
             if (Mail::isEnabled()) {
                 try {
-                    $verify_token = App::getInstance(true)->generateCode();
-                    $appInstance->getLogger()->debug('Verify token: ' . $verify_token);
-                    Verification::add($verify_token, $uuid, EmailVerificationColumns::$type_verify);
-                    $appInstance->getLogger()->debug('Verification added');
-                    Verify::sendMail($uuid, $verify_token);
-                    $appInstance->getLogger()->debug('Email sent');
+                    if ($config->getDBSetting(ConfigInterface::FORCE_MAIL_LINK, false)) {
+                        $verify_token = App::getInstance(true)->generateCode();
```

PHPMD (2.15.0)

The variable $verify_token is not named in camelCase. (Controversial Rules)

(CamelCaseVariableName)